From 220b17102fda18b4896fb5fca1c79fcff6512fc6 Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Mon, 20 Aug 2018 22:57:43 +0300
Subject: [PATCH 166/296] SystemUI: No Signal bar displays for a moment in UI
 after Bootup

No signal bar should be not displayed when airplaneMode is enabled
---
 .../SystemUI/src/com/android/systemui/statusbar/SignalClusterView.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/SignalClusterView.java b/packages/SystemUI/src/com/android/systemui/statusbar/SignalClusterView.java
index 7df3e988acb..3d46e5d0325 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/SignalClusterView.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/SignalClusterView.java
@@ -546,7 +546,7 @@ public class SignalClusterView extends LinearLayout implements NetworkController
             mWifiSignalSpacer.setVisibility(View.GONE);
         }
 
-        if (mNoSimsVisible) {
+        if (mNoSimsVisible && !mIsAirplaneMode) {
             mIconLogger.onIconShown(SLOT_MOBILE);
             mNoSimsCombo.setVisibility(View.VISIBLE);
             if (!Objects.equals(mSimDetected, mNoSimsCombo.getTag())) {
-- 
2.11.0

